src/main/java/com/minecolonies/coremod/commands/colonycommands/RefreshColonyCommand.java

```diff
@@ -106,6 +152,21 @@ public void execute(@NotNull final MinecraftServer server, @NotNull final IComma
             return;
         }
 
+        executeShared(server, sender, colony);
+    }
+
+    private void executeShared(@NotNull final MinecraftServer server, @NotNull final ICommandSender sender, @NotNull final Colony colony)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "server". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1172)